child_compassion/demo/compassion.child.csv

```diff
@@ -0,0 +1,67 @@
+id,code,name
```

Comment:
name is not a mandatory field and is automatically populated by the webservice when record is created, so I think you can remove this column.